manifest.json

```diff
@@ -31,7 +31,7 @@
         {
           "platforms": [
             {
-              "dockerfile": "src/alpine/3.6-WithNode",
+              "dockerfile": "src/alpine/3.6/amd64/WithNode",
```

Comment:
The architecture dir should always be what contains the Dockerfile.

src/alpine/3.6
---------------amd64
---------------withnode/amd64
---------------forhelix/amd64

This will make the pattern consistent and align with the tag naming pattern.

Reply:
ok ptal I believe this should be addressed